### DISCUSSION TITLE
Maintenance

### DISCUSSION THREAD

**Discussion by reichhartd** [orval-labs/orval#574] [General]
Hello @anymaniax  👋

Would it be possible to become a maintainer of Orval as well?

We now rely on a secure generation of Axios functions, React Query hooks and mocks for our test cases with two projects. This would make it easier for us to react faster when bugs occur, and also support the [development of the new website](https://github.com/anymaniax/orval/discussions/529).

Kind regards from Berlin

**Comment by anymaniax** [comment]
Hello @reichhartd,

That would be great. I would like to know you better before that and put some stuff in place.

I have personal issues at the moment. Everything should be fixed at the end of this month. 

I will contact you to how we can manage to do that properly.

Kind regards from Belgium

**Comment by anymaniax** [comment]
Hello @reichhartd, still interested in being a maintainer? Sorry, the last couple of month was complicated. I would need some support to improve `orval`. It's a bit my baby and I don't always have the time to take care of it 😅 and improve it.

**Reply by reichhartd** [reply]
As I described at https://github.com/anymaniax/orval/discussions/506#discussioncomment-5100468, we are not working with Orval anymore because we have switched from REST to GraphQL.
Therefore, it does not make sense to me at the moment. If this should change, I will let you know.
